en/get-started.md

```diff
@@ -196,6 +196,14 @@ minikube start --docker-env https_proxy=http://127.0.0.1:1086 \
 
 See [minikube setup](https://kubernetes.io/docs/setup/minikube/) for more options to configure your virtual machine and Kubernetes cluster.
 
+To interact with the cluster, you can use `kubectl`. This is included as a sub-command in `minikube`. The alias below can be used to make this available directly as the rest of the documentation expects. You will have to put this in your shell profile or execute this after opening a new shell.
```

Comment:
There are four "This" used in the paragraph. To make the content easier to understand, could we rephrase as below? Thanks!

```suggestion
To interact with the cluster, you can use `kubectl`, which is included as a sub-command in `minikube`. To make `kubectl` take effect as the rest of the documentation expects, you can either add the following alias definition command to your shell profile or execute the following alias definition command after opening a new shell.
```